UnitTests/BraintreePayPalTests/BTPayPalClient_Tests.swift

```diff
@@ -1028,4 +1028,20 @@ class BTPayPalClient_Tests: XCTestCase {
 
         XCTAssertFalse(mockAPIClient.postedIsVaultRequest)
     }
+    
+    func testTokenize_whenCheckoutRequest_setSessionID() async {
```

Comment:
Maybe we could add a test where we validate the sessionID before and after, essentially to test the overwriting. What do you think?

```
XCTAssertEqual(mockAPIClient.metadata.sessionID, "some-previous-value")

 let checkoutRequest = BTPayPalVaultRequest()
 checkoutRequest.shopperSessionID = "test-shopper-insights-id"
  let _ = try? await payPalClient.tokenize(checkoutRequest)

XCTAssertEqual(mockAPIClient.metadata.sessionID, "overwriten-value")
```